torch/distributed/_composable/fsdp/_fsdp_init.py

```diff
@@ -0,0 +1,18 @@
+from typing import Union
+
+import torch
+
+
+def _normalize_device(device: Union[torch.device, int, str]) -> torch.device:
```

Comment:
We have a type Alias for that here: https://github.com/pytorch/pytorch/blob/af831415a876079a2ff63d72ae99d03e4c11aebd/torch/_prims_common/__init__.py#L55

Reply:
I changed it to use this type alias. 

The rebase conflicts when changing the imports (e.g. touching `from typing import ...`) are rough 😢 .